Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+                                  json_data=data)
+
+    def custom_whitelist_delete_request(self, id: str) -> Dict[str, Any]:
+        """Delete a custom whitelist member.
+
+        Args:
+            id (str): Delete a custom whitelist member.
+
+        Returns:
+            Dict[str, Any]: API response from FortiwebVM V1
+        """
+        endpoint = f'ServerObjects/Global/CustomGlobalWhiteList/{id}'
+        return self._http_request(method='DELETE', url_suffix=endpoint)
+
+    def geo_exception_list_request(self) -> Dict[str, Any]:
+        """List the Geo IP Exception.
+
+        Returns:
+            Dict[str, Any]: API response from FortiwebVM V1
+        """
+        return self._http_request(method='GET', url_suffix='WebProtection/Access/GeoIPExceptionsList')
+
+    def trigger_policy_list_request(self) -> Dict[str, Any]:
+        """List the Trigger Policies.
+
+        Returns:
+            Dict[str, Any]: API response from FortiwebVM V1
+        """
+        return self._http_request(method='GET', url_suffix='LogReport/LogPolicy/TriggerList')
+
+    def custom_predifined_whitelist_list_request(self) -> Dict[str, Any]:
+        """List the Custom Predifined members.
+
+        Returns:
+            Dict[str, Any]: API response from FortiwebVM V1
+        """
+        return self._http_request(method='GET', url_suffix='ServerObjects/Global/CustomPredefinedGlobalWhiteList')
+
+    def custom_predifined_whitelist_update_request(self, data: Dict[str, Any]) -> Dict[str, Any]:
+        """Update a Custom Predifined members.
+
+        Returns:
+            Dict[str, Any]: API response from FortiwebVM V1
+        """
+        response = self._http_request(method='PUT',
+                                      url_suffix='ServerObjects/Global/CustomPredefinedGlobalWhiteList',
+                                      json_data=data)
+        return response
+
+    def certificate_intermediate_group_list_request(self) -> Dict[str, Any]:
+        """List the Certificate intermediate groups.
+
+        Returns:
+            Dict[str, Any]: API response from FortiwebVM V1
+        """
+        return self._http_request(method='GET', url_suffix='System/Certificates/InterCAGroupList')
+
+
+class ClientV2(Client):
+    """Fortiweb VM V2 Client
+
+    Args:
+        Client (Client): Client class with abstract functions.
+    """
+    API_VER = 'V2'
+
+    def __init__(self, base_url: str, api_key: str, version: str, proxy: bool, verify: bool):
+        super().__init__(base_url=base_url,
+                         api_key=api_key,
+                         version=version,
+                         endpoint_prefix='api/v2.0/',
+                         verify=verify,
+                         proxy=proxy)
+
+    @property
+    def not_exist_error_list(self) -> List[int]:
+        """Sends not exists errors in Fortiweb V2.
+
+        Returns:
+            List[int]: Not exists errors in Fortiweb V2.
```

Comment:
please clarify. e.g. `Error codes for when values do not exist in the system`
Please check other docstrings as well